prettier/prettier-check.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: prettier-check-from-gitrepo
+spec:
+  workspaces:
+  - name: source
+    mountPath: /source
+  params:
+  - name: subdirectory
+    type: string
+    description: Subdirectory inside "source" workspace that contains the target files.
+    default: "."
+  - name: filepath
```

Comment:
Looking at the docs, it looks like you can provide multiple paths:

```
prettier [options] [file/dir/glob ...]
```

We may want to make some small changes to allow multiple paths:
1. rename `options` to `args`, default it to `--check .`
2. set `workingDir` to the workspace path (this way all paths can be relative to the workspace)